aries-site/src/data/structures/templates.js

```diff
@@ -108,6 +108,15 @@ export const templates = [
         fit: 'contain',
       },
     },
+    relatedContent: [
```

Comment:
Can we update `available: false` to be `available: true`? This will allow PageLayouts card to be prioritized in the card sort displays.

Reply:
Good call, thanks.